.github/ISSUE_TEMPLATE/cut_release.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Cut Release
+about: Create a new release checklist
+title: 'Cut {VERSION}'
+labels: 'kind/release'
+assignees: ''
+
+---
+
+**Summary:**
+Task covering patch release work.
+
+Dev Complete: {RELEASE_DATE} (Typically ~1 week prior to upstream release date)
+
+**List of required releases:**
+_To release as soon as able for QA:_
+- {VERSION}
+
+_To release once have approval from QA:_
+- {VERSION} (Never release on a Friday unless specified otherwise)
+
+**Prep work:**
+- [ ] PM: Dev and QA team to be notified of the incoming releases - send calendar invite
+- [ ] PM: Dev and QA team to be notified of the date we will mark the latest release as stable - send calendar invite
```

Comment:
Two things below.

Nit:
```suggestion
- [ ] PM: Dev and QA team to be notified of the date we will mark the latest release as stable - add event to team calendar
```

Note:
Only the latest minor release would have this checklist item (e.g. 1.21.x currently). So, do we need to qualify this or something? Basically this checklist item does not apply to older minor release branches like 1.19 and 1.20 today.